### PR TITLE
Feat: Add card to landing page

### PR DESCRIPTION
**Pull request by lilinkw**

### CODE REVIEW

File: src/components/ui/card/card-info.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  subTitle?: string
+  titleProps?: TTitleProps
+  headerProps?: THeaderProps
+  subTitleProps?: TSubtitleProps
+}
+
+export function CardInfo(props: ICardInfoProps) {
+  const { title, subTitle, titleProps } = props
+  return (
+    <Card className="min-w-[100px]">
```
Comment (blackmouse572) [inline src/components/ui/card/card-info.tsx:22]: Seem like this card is not round enough 
image

File: src/components/ui/card/card-with-form.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import { Label } from "@/components/ui/label"
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select"
+
+export function CardWithForm() {
```
Comment (blackmouse572) [inline src/components/ui/card/card-with-form.tsx:22]: Preview or implement please !

File: src/components/ui/card/card-info.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <CardHeader>
+        <CardTitle ref={titleProps?.ref} {...titleProps?.props}>
+          {title}
+        </CardTitle>
+        <CardDescription>{subTitle}</CardDescription>
+      </CardHeader>
```
Comment (blackmouse572) [inline src/components/ui/card/card-info.tsx:28]: Missing Icon ?

**Review by blackmouse572 — CHANGES_REQUESTED** [review]
Need t dynami

File: src/components/ui/card/card-with-form.tsx (hunk 2 above)
Comment (lilinkw) [reply src/components/ui/card/card-with-form.tsx:22]: What do you mean by preview? is it to display on UI?

Comment (blackmouse572) [reply src/components/ui/card/card-with-form.tsx:22]: yes, any. A picture or a direction to see the component

**Review by blackmouse572 — CHANGES_REQUESTED** [review]

File: src/components/ui/card/card-with-form.tsx (hunk 2 above)
Comment (lilinkw) [reply src/components/ui/card/card-with-form.tsx:22]: ok. Try show form button, just an e.g. Need modification later

**Comment by lilinkw** [comment]
@blackmouse572

**Comment by lilinkw** [comment]
image
@blackmouse572

File: src/app/[locale]/(main)/page.tsx [hunk 4]
```diff
@@ -15,16 +60,45 @@ export default function Home() {
...
         </p>
         <p className="relative z-20 bg-gradient-to-b from-slate-300 to-slate-700 bg-clip-text text-sm font-black tracking-wider  text-transparent sm:text-base ">
           Bring joy and fun to your classroom
         </p>
+        <LoginCard />
       </div>
 
-      <div className="space-y-4 text-center">
+      {/* Recent and stats */}
+      <div className="bg-[#E5E5E5] p-10">
```
Comment (blackmouse572) [inline src/app/[locale]/(main)/page.tsx:74]: use `bg-neutral`

File: src/app/[locale]/(main)/page.tsx [hunk 5]
```diff
@@ -15,16 +60,45 @@ export default function Home() {
...
+                className="h-[14vh] max-h-[20vh] w-[21.9vw] max-w-[40vw]"
+                key={i}
+                {...card}
+              />
+            ))}
+          </div>
+        </div>
+        <div className="p-2">
+          <div className="mb-2 underline">Stats</div>
+          <div className="flex justify-between space-x-4">
```
Comment (blackmouse572) [inline src/app/[locale]/(main)/page.tsx:89]: flex cannot goes with `space-x-4`, use `gap`

File: src/components/ui/card/card-info.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+  return (
+    <Card
+      {...others}
+      className={cn("min-w-[100px] rounded-3xl", others.className)}
+    >
+      <CardHeader
+        {...headerProps}
+        className={cn(
+          "flex flex-row justify-between space-x-2 p-5",
```
Comment (blackmouse572) [inline src/components/ui/card/card-info.tsx:52]: flex cannot goes with space-x-2 use gap instead

File: src/components/ui/card/card-info.tsx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+        )}
+      >
+        <div className="space-y-2 overflow-hidden text-ellipsis">
+          <CardTitle {...titleProps}>{title}</CardTitle>
+          <CardDescription className="truncate" {...subTitleProps}>
+            {subTitle}
+          </CardDescription>
+        </div>
+        {arrowIcon && (
+          <div className="h-[50px] w-[50px] rounded-full bg-[#E5E5E5]">
```
Comment (blackmouse572) [inline src/components/ui/card/card-info.tsx:63]: use bg-neutral

File: src/app/[locale]/(main)/page.tsx [hunk 8]
```diff
@@ -15,16 +60,45 @@ export default function Home() {
...
 
-      <div className="space-y-4 text-center">
+      {/* Recent and stats */}
+      <div className="bg-[#E5E5E5] p-10">
+        <div className="p-2">
+          <div className="mb-2 underline">Recent</div>
+          <div className="flex justify-between space-x-4">
+            {cardInfos.map((card, i) => (
+              <CardInfo
+                className="h-[14vh] max-h-[20vh] w-[21.9vw] max-w-[40vw]"
```
Comment (blackmouse572) [inline src/app/[locale]/(main)/page.tsx:80]: This not working on many screen size.  Maybe set a ratio or use strict values to keep shape 
Examples
On large screen:
image
on less height screen
image

File: src/app/[locale]/(main)/page.tsx [hunk 9]
```diff
@@ -15,16 +60,45 @@ export default function Home() {
     <div className="min-h-screen">
       <div className="relative flex h-[70vh] w-full flex-col items-center justify-center overflow-hidden">
         <div className="pointer-events-none absolute inset-0 z-10 h-full w-full bg-background [mask-image:linear-gradient(to_top_right,white_40%,transparent_50%)]" />
-        <Boxes row={60} className="to-[-30%]" />
+        <Boxes row={60} className="-top-[30%]" />
```
Comment (blackmouse572) [inline src/app/[locale]/(main)/page.tsx:63]: Eslint warning -> use top-[-30%]

**Review by blackmouse572 — CHANGES_REQUESTED** [review]
image
image

**Pull request closed without merging**